packages/emails/src/templates/BaseScheduledEmail.tsx

```diff
@@ -20,7 +20,7 @@ export const BaseScheduledEmail = (
     calEvent: CalendarEvent;
     attendee: Person;
     timeZone: string;
-    includeAppsStatus?: boolean;
+    isOrganizer?: boolean;
```

Comment:
App status is only included for organizers. Seemed unnecessary to add another prop.